HTML/Git.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+
+<body style="font-family: Vemana2000, sans-serif;">
+
+    <ul style="list-style-type: none; overflow: hidden; background-color: black; margin-left: 0%; padding-top:25px; padding-bottom: 10px;">
+        <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+           <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e0/Git-logo.svg/1024px-Git-logo.svg.png" width="70px;">
+        </li>
+        <li class="li" style="display: block; float: left; margin-right:27px; font-size:20px;">
+            <a href="Git.html" style="color:white; text-decoration: none;">Git</a>
+        </li>
+        <li style="display: block; float: left; margin-right:27px; ">
+            <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+        </li>
+        <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+            <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+        </li>
+        <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+            <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+        </li>
+    </ul>
+<h2>
+    Git
+</h2>
+<div>
+    Git ist ein beliebtes VCS und wurde 2005 von Linus Torvalds, dem Entwickler hinter Linux, erstellt.<br>
+    Es konnte sich aufgrund seiner hohen Sicherheit und Effizienz durchsetzen und Verteilte, gleichzeitig ablaufende Arbeitsvorgänge.
+    Die Versionskontrolle geschieht hier durch das Branch System. Will ein<br> Entwickler z.B. änderungen an einem Code machen kann dieser eine Kopie von der neusten Version machen und dann dieses in einen unabhängigen Branch bearbeiten und testen.
+    <br>Diese änderungen werden zuallererst im Lokalen Repository gespeichert und werden dann dem Team per pull request zur Verfügung gestellt.<br>
+    Git kann und wird dabei häufig zusammen mit GitHub verwendet. Es dient hierbei als remote repository und erleichtert entwicklern die Zusammenarbeit.</div> <img src="https://de.wizcase.com/wp-content/uploads/2022/03/GitHub-Logo.png" width="200px;"><br>
+<div>Das Programm selber wird mithilfe von den Git kommandos kontrolliert:</div>
+
+
```

Comment:
:hammer_and_wrench: Diese Format ist besser lesbar
```suggestion
<div>
    Git ist ein beliebtes VCS und wurde 2005 von Linus Torvalds, dem Entwickler hinter Linux, erstellt.<br>
    Es konnte sich aufgrund seiner hohen Sicherheit und Effizienz durchsetzen und Verteilte, gleichzeitig ablaufende
    Arbeitsvorgänge.
    Die Versionskontrolle geschieht hier durch das Branch System. Will ein<br> Entwickler z.B. änderungen an einem Code
    machen kann dieser eine Kopie von der neusten Version machen und dann dieses in einen unabhängigen Branch bearbeiten
    und testen.
    <br>Diese änderungen werden zuallererst im Lokalen Repository gespeichert und werden dann dem Team per pull request
    zur Verfügung gestellt.<br>
    Git kann und wird dabei häufig zusammen mit GitHub verwendet. Es dient hierbei als remote repository und erleichtert
    entwicklern die Zusammenarbeit.
</div>
<img alt="github-logo" src="https://de.wizcase.com/wp-content/uploads/2022/03/GitHub-Logo.png" width="200px;"><br>
<div>
    Das Programm selber wird mithilfe von den Git kommandos kontrolliert:
</div>

```